Makefile.inc

```diff
@@ -65,7 +65,11 @@ endif
 
 ifeq ($(WINBASED),yes)
 EXT        = .exe
+  ifneq (,$(filter MINGW%,$(TARGET_OS)))
+# WINDRES should already be set correctly for MinGW
+  else
 WINDRES    = windres
```

Comment:
If the goal is to not redefine `WINDRES` when it's already defined by environment,
what about : 
``` 
WINDRES ?= windres
```

Reply:
Thanks for pointing this out. I was too narrowly focused on solving the problem at hand.